Mlem/Views/Shared/Links/Community Link View.swift

```diff
@@ -135,11 +135,15 @@ struct CommunityLabel: View {
         serverInstanceLocation == .bottom ? AppConstants.largeAvatarSize : AppConstants.smallAvatarSize
     }
     
+    private func avatarUrl(from: URL) -> URL {
```

Comment:
nit

Can we do;
```swift
avatarUrl(from url: URL) -> URL {
    serverInstanceLocation == .bottom ? url.withIcon64Parameters : url.withIcon32Parameters
}
```

it lets the function use `url` internally for the argument name instead of `from`

Reply:
What's the significance of the `from` keyword in that case? I've seen it around but don't get what it does

Reply:
It's just semantics I guess, you can have a public name and an internal name, in the case of the function here it lets you write at the call site:

`avatarUrl(from: ...)`

but then inside the function you can refer to the argument as `url` which makes sense, rather than having to call it `from` which doesn't really give a clue as to what it is